service-front/module/Application/src/Services/OpgApiService.php

```diff
@@ -163,14 +163,115 @@ public function createCase(
         string $lastname,
         string $dob,
         string $personType,
-        array $lpas
+        array $lpas,
+        array $address,
     ): array {
         return $this->makeApiRequest("/cases/create", 'POST', [
             'firstName' => $firstname,
             'lastName' => $lastname,
             'dob' => $dob,
             'personType' => $personType,
             'lpas' => $lpas,
+            'address' => $address
         ]);
     }
+
+    public function updateIdMethod(string $uuid, string $method): void
+    {
+        $data = [
+            'idMethod' => $method
+        ];
+        try {
+            $this->makeApiRequest("/cases/$uuid/update-method", 'POST', $data);
+        } catch (\Exception $exception) {
+            throw new OpgApiException($exception->getMessage());
+        }
+    }
+
+    public function listPostOfficesByPostcode(string $uuid, string $postcode): array
+    {
+//        $data = [
+//            'postcode' => $postcode
+//        ];
+//        try {
+//            $response = $this->makeApiRequest("/cases/$uuid/find-post-office", 'POST', $data);
+//            return $response['result'];
+//        } catch (OpgApiException $opgApiException) {
+//            throw new OpgApiException('Post office reference code not found');
+//        }
+
+        return [
+            12345672 => [
+                'name' => 'Hednesford',
+                'address' => '45 Market Street, Hednesford, Cannock, WS12 1AY',
+                'distance' => '0.5 miles'
```

Comment:
I still think we shouldn't include `distance` here because we won't be able to get it from the yoti API. I'll raise this with Matthew